device-types/DataDirect Networks/SFA200NVX2-Enclosure.yaml

```diff
@@ -0,0 +1,72 @@
+---
+manufacturer: DataDirect Networks
+model: SFA200NVX2 Enclosure
+slug: datadirect-networks-sfa200nvx2-enclosure
+comments: 24bay Enclosure for SFA200NVX2 or SFA400NVX2
+part_number: ENC-200NVX2
+u_height: 2
```

Comment:
Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

Reply:
add done